### PR TITLE
Saved views and more

### PR DESCRIPTION
**Pull request by thoragudf**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…/start-using-inspector**](https://docs-jg9vcgk6l.avo-dev.app/data-design/start-using-inspector) | [**…-events-from-inspector**](https://docs-jg9vcgk6l.avo-dev.app/workspace/add-events-from-inspector) |
|:-:|:-:|
|<a href="https://docs-jg9vcgk6l.avo-dev.app/data-design/start-using-inspector">Screenshot of /data-design/start-using-inspector</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/egRjR9nPwRwB2TeoWAtDDrVLkuk88-EdCuFjqA-rEQYsGkUsAnq7Ll-q0XCZ-ELQsjllPRdFUzq16CRAJIbEAA.png">(view full size)</a> |<a href="https://docs-jg9vcgk6l.avo-dev.app/workspace/add-events-from-inspector">Screenshot of /workspace/add-events-from-inspector</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/I4JeEHE7b1CfZcaF0UTaoqkco4O4Z6T3-RPrK0LJPL2bm3k04GfoCL_XkQBULg5dCkrxOpjBnC2BZ0VFsZCTrw.png">(view full size)</a> |



| [**…ring-inspector-sources**](https://docs-jg9vcgk6l.avo-dev.app/workspace/inspector/configuring-inspector-sources) | [**…/inspector-issues-view**](https://docs-jg9vcgk6l.avo-dev.app/workspace/inspector/inspector-issues-view) |
|:-:|:-:|
|<a href="https://docs-jg9vcgk6l.avo-dev.app/workspace/inspector/configuring-inspector-sources">Screenshot of /workspace/inspector/configuring-inspector-sources</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/VS82boedhlikVMsh6Ipv3RLA2m3oiPIZqE2H6HbZ2j5qcDEk3yO0DvtCgdeP1f5uLQaF3Zpk9dz1oe9-Ad1mQg.png">(view full size)</a> |<a href="https://docs-jg9vcgk6l.avo-dev.app/workspace/inspector/inspector-issues-view">Screenshot of /workspace/inspector/inspector-issues-view</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/6tbfR1sTRrwRubZkWNHYoJ3qvbb_2Js1JJlh6jlg7u2U8QpNcgPCfZe4nwDtrPUZvsmH--eRd5h0e8eIrO1a2A.png">(view full size)</a> |



| [**…inspector-slack-alerts**](https://docs-jg9vcgk6l.avo-dev.app/workspace/inspector/inspector-slack-alerts) | [**…acking-plan/properties**](https://docs-jg9vcgk6l.avo-dev.app/workspace/tracking-plan/properties) |
|:-:|:-:|
|<a href="https://docs-jg9vcgk6l.avo-dev.app/workspace/inspector/inspector-slack-alerts">Screenshot of /workspace/inspector/inspector-slack-alerts</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/YwULsFdoFn--ZClP_mDF3Pjp6pygt7PTlHBlMjXPm1HomgFBidUKpjNE3cRHldI1ztEpIOjok0pdXOc2G1gGtw.png">(view full size)</a> |<a href="https://docs-jg9vcgk6l.avo-dev.app/workspace/tracking-plan/properties">Screenshot of /workspace/tracking-plan/properties</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/W0G7vax3jmLPkAlrAPL5NM_6PEKpmQrWDxk6_2Ia3T0IdYqfpwH0pTSG9gbFYqNGnWIMTpLBI_4rhkgYEGpE-g.png">(view full size)</a> |







Commit fac99c4d4f86dba62c6ca3245b1e23454d011684 (https://docs-jg9vcgk6l.avo-dev.app).

File: .spelling [hunk 1]
```diff
@@ -262,6 +262,7 @@ unarchiving
 unarchived
 unbundle
 unbundling
+yay
```
Comment (hugihlynsson) [inline .spelling:265]: 🙌

File: images/inspector/add-event-from-events-view.png
Comment (hugihlynsson) [inline images/inspector/add-event-from-events-view.png:1]: We need to minify this picture along with all the other. https://tinypng.com/ is my fav for this

File: pages/data-design/start-using-inspector.mdx [hunk 2]
```diff
@@ -34,13 +34,27 @@ Inspector does not process property values or PII data, only event names, proper
 
 TL;DR: No PII data ever flows through Avo servers; only event schemas.
 
-## How to use Inspector
+## Learning about Inspector issues
 
 After <Link passHref href="/workspace/inspector#configuring-sources"><a>setting up your sources</a></Link> to send data to the Inspector with the direct CDP integrations or <Link passHref href="/implementation/avo-inspector-sdk-reference"><a>Avo Inspector SDKs</a></Link> you will see the data appearing in the the Inspector tab.
+There are two overviews available Inspector your incoming data and issues, the <Link passHref href="/workspace/inspector/inspector-issues-view"><a>Issues view</a></Link> and the <Link passHref href="/workspace/inspector/inspector-events-view"><a>Events view</a></Link> – and you can set up <Link passHref href="/workspace/inspector/inspector-slack-alerts"><a>Slack Alerts</a></Link> to be notified about new issues.
```
Comment (hugihlynsson) [inline pages/data-design/start-using-inspector.mdx:40]: > There are two overviews available Inspector your incoming data and issues [...]

Does this pass as a valid sentance? It reads a bit weird to me.

File: pages/workspace/add-events-from-inspector.mdx [hunk 3]
```diff
@@ -5,33 +5,75 @@ title: 'Add Events and properties from Inspector'
 
 import Link from '../../components/Link';
 
-## Inspector Dashboard
+The Inspector compares the events and properties coming in to the tracking plan and detects if there are any of the incoming missing from the tracking plan. Event and properties can both be added to the tracking plan from the <Link passHref href="/workspace/inspector/inspector-issues-view"><a>Issues view</a></Link> and the <Link passHref href="/workspace/inspector/inspector-events-view"><a>Events view</a></Link>.
```
Comment (hugihlynsson) [inline pages/workspace/add-events-from-inspector.mdx:8]:
```suggestion
The Inspector compares the injested events and properties against the tracking plan and detects if there are any properties missing from the tracking plan. Unknown events and properties can be added to the tracking plan from both the <Link passHref href="/workspace/inspector/inspector-issues-view"><a>Issues view</a></Link> and the <Link passHref href="/workspace/inspector/inspector-events-view"><a>Events view</a></Link>.
```

File: pages/workspace/add-events-from-inspector.mdx [hunk 4]
```diff
@@ -5,33 +5,75 @@ title: 'Add Events and properties from Inspector'
...
+
+## Event not in Tracking Plan
+
+### Issues view
+
+When an event is missing from the tracking plan, the issue details of the issue view will offer you to add it to the tracking plan right from there. Click the "Add to Tracking Plan" button and the event and all it's properties are created.
```
Comment (hugihlynsson) [inline pages/workspace/add-events-from-inspector.mdx:14]:
```suggestion
When an event is missing from the tracking plan the issue details view will offer you to quickly add it to the tracking plan. Click the "Add to Tracking Plan" button and the event and all it's properties are created.
```
Although this is not true today, is it? We're not displaying issuse for unknown events?

File: pages/workspace/add-events-from-inspector.mdx [hunk 5]
```diff
@@ -5,33 +5,75 @@ title: 'Add Events and properties from Inspector'
...
     width="100%"
-    alt="Image shows how to change property presence between events"
+    alt='Image shows issue details with "Add to tracking plan" highlighted for the event.'
   />
 </center>
 
-The inspector shows you all events and properties it detects, and lets you know whether they are missing from your tracking plan, and gives you the option to add the event or property to your tracking plan directly from the Inspector dashboard.
+### Events view
+
+If the Inspector detects an Event that is not in your tracking plan, it will be highlighted in the events view with a red tracking plan icon next to the name, along with an "Add" button to add it to your tracking plan. The slide-over in the events view also has an option to add the event right from there. Click either button to add the event and all it's properties to the tracking plan.
```
Comment (hugihlynsson) [inline pages/workspace/add-events-from-inspector.mdx:26]:
```suggestion
If the Inspector detects an event that is not in your tracking plan, it will be highlighted with a red tracking plan icon next to the name, along with an "Add" button to add it to your tracking plan. The events view sidebar also has an option to add the event right from there. Click either button to add the event and all it's properties to the tracking plan.
```

File: pages/workspace/add-events-from-inspector.mdx [hunk 6]
```diff
@@ -5,33 +5,75 @@ title: 'Add Events and properties from Inspector'
...
+  <img
+    src={require('../../images/inspector/add-property-from-issue-details.png')}
+    width="100%"
+    alt='Image shows issue details with "Add to tracking plan" highlighted for the property.'
+  />
+</center>
+
+### Events view
+
+Selecting an event in the events view will expand down such that all the properties that were detected in this particular event are shown. If a property is missing from the tracking plan for that particular event, it is highlighted with a red tracking plan icon next to the property name, along with an "Add" button to add it to your tracking plan. The slide over in the events view also has an option to add the property to the tracking right from there. Click either button to add the event and all it's properties to the tracking plan. There are two cases for adding property to the event in the tracking plan:
```
Comment (hugihlynsson) [inline pages/workspace/add-events-from-inspector.mdx:62]:
```suggestion
Clicking no the event name cell will expand it such that all the properties that were detected in this particular event are shown. If a property is missing from the tracking plan for that particular event, it is highlighted with a red tracking plan icon next to the property name, along with an "Add" button to add it to your tracking plan. The events view sidebar also has an option to add the property to the tracking right from there. Click either button to add the event and all it's properties to the tracking plan. There are two cases for adding property to the event in the tracking plan:
```

File: pages/workspace/inspector/inspector-slack-alerts.mdx [hunk 7]
```diff
@@ -6,18 +6,33 @@ abstract: 'How to start recieving Slack alerts for your Inspector issues'
 
 import Link from '../../../components/Link';
 
-In the Inspector events view you can connect Inspector to your Slack to receive alerts when new issues appear in your production environments.
+You can connect Inspector to your Slack to receive alerts when new issues appear in your production environments.
```
Comment (hugihlynsson) [inline pages/workspace/inspector/inspector-slack-alerts.mdx:9]:
```suggestion
Connect Inspector to your Slack to receive alerts when new issues appear in your production environments.
```

**Review by hugihlynsson — COMMENTED** [review]
Great start!

File: pages/data-design/start-using-inspector.mdx (hunk 2 above)
Comment (thoragudf) [reply pages/data-design/start-using-inspector.mdx:40]: haha you are so picky Hugi

File: images/inspector/add-event-from-events-view.png
Comment (thoragudf) [reply images/inspector/add-event-from-events-view.png:1]: ojojoj I forgot about that small detail. Will do

File: pages/workspace/add-events-from-inspector.mdx (hunk 4 above)
Comment (thoragudf) [reply pages/workspace/add-events-from-inspector.mdx:14]: We're not displayin the issue for missing events, but you can still add the event from the issue details if it is not in the tracking plan.

**Comment by thoragudf** [comment]
Jæja... this is finally ready for proper view. Although I'm wondering if saved views deserve their own page? What do you think @hugihlynsson @bragur @klaraavo ?

**Review by bragur — APPROVED** [review]

File: pages/data-design/start-using-inspector.mdx (hunk 2 above)
Comment (thoragudf) [reply pages/data-design/start-using-inspector.mdx]: Ok done!

**Review by hugihlynsson — APPROVED** [review]
